plugins/Jobs.cpp

```diff
@@ -1000,6 +985,29 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
             STHROW("502 Failed to update job data");
         }
 
+        // If this was the last queued child, resume the parent
+        SQResult result;
+        if (!db.read("SELECT parentJobID "
+                     "FROM jobs "
+                     "WHERE jobID=" + SQ(jobID) + ";",
+                     result)) {
+            STHROW("502 Select failed");
+        }
+        const string& safeParentJobID = SQ(result[0][0]);
+        if (!db.read("SELECT count(1) "
+                     "FROM jobs "
+                     "WHERE parentJobID=" + safeParentJobID + " AND "
+                       "state IN ('QUEUED', 'RUNQUEUED');",
```

Comment:
You basically need to add RUNNING here too, otherwise, like you mention, you will resume the parent with RUNNING children and you don't want that, just let the RUNNING child resume the parent.

Reply:
🤦‍♂️